Symfony/CS/Fixer/Symfony/AliasFunctionsFixer.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author Vladimir Reznichenko <kalessil@gmail.com>
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+class AliasFunctionsFixer extends AbstractFixer
+{
+    /**
+     * @var string[]
+     */
+    private static $aliases = array(
+        'is_double' => 'is_float',
+        'is_integer' => 'is_int',
+        'is_long' => 'is_int',
+        'is_real' => 'is_float',
+        'sizeof' => 'count',
+        'doubleval' => 'floatval',
+        'fputs' => 'fwrite',
+        'join' => 'implode',
+        'key_exists' => 'array_key_exists',
+        'chop' => 'rtrim',
+        'close' => 'closedir',
+        'ini_alter' => 'ini_set',
+        'is_writeable' => 'is_writable',
+        'magic_quotes_runtime' => 'set_magic_quotes_runtime',
+        'pos' => 'current',
+        'rewind' => 'rewinddir',
+        'show_source' => 'highlight_file',
+        'strchr' => 'strstr',
+    );
+
+    /**
+     * @return string[]
+     */
+    public static function getAliases()
+    {
+        return self::$aliases;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, Tokens $tokens)
+    {
+        foreach ($tokens->findGivenKind(T_STRING) as $index => $token) {
```

Comment:
More info needed here - what the goal for making changes (I familiar with continuation and early returns).